docs/getting-started.mdx

```diff
@@ -105,15 +105,54 @@ They are documented in the [API Reference](pathname:///api/v7/classes/Sequelize.
 
 ### Testing the connection
 
-You can use the `.authenticate()` function to test if the connection is OK:
+You can use the `.authenticate()` function to test if the connection is OK, it runs `SELECT 1+1 AS result` query to test:
```

Comment:
I don't think we should document here what `authenticate` executes. The query is dialect dependant and a dialect could decide to do something completely different. It's not very useful information.

That being said I think we should consider deprecating `sequelize.authenticate()` in favor of `sequelize.testDatabaseConnection()` or some better name. It would make its purpose clearer, it's not an authentication tool.